src/serlio/utils/Utilities.h

```diff
@@ -96,6 +96,21 @@ std::vector<const C*> toPtrVec(const std::vector<std::unique_ptr<C, D>>& sv) {
 	return pv;
 }
 
+template <class SizeT>
```

Comment:
maybe add a comment with a link to the boost implementation

Reply:
added comment with link